sonic_platform_base/module_base.py

```diff
@@ -77,6 +89,83 @@ def get_system_eeprom_info(self):
         """
         raise NotImplementedError
 
+    def get_name(self):
+        """
+        Retrieves the name of the module prefixed by CONTROL-CARD, LINE-CARD,
+        FABRIC-CARD
+
+        Returns:
+            string: A string providing the name of the card prefixed by one of the
+            MODULE_TYPE_CONTROL, MODULE_TYPE_LINE, MODULE_TYPE_FABRIC followed by
+            an index.
+            Ex. A Chassis having 1 control-card, 4 line-cards and 6 fabric-cards
+            can provide names CONTROL-CARD1, LINE-CARD1 to LINE-CARD4,
```

Comment:
CONTROL-CARD1 -> SUPERVISOR1

Reply:
Since we decided to start card/module indexing with '0', can we change this to SUPERVISOR0.
Also, plan is to insert Supervisor/control-card in slot0 if vendor/platform has support of inserting two Supervisors in the Modular chassis.
Tomorrow, peer-Supervisor would be taken as SUPERVISOR1

Reply:
Fixed.